projects/admin/src/app/record/detail-view/template-detail-view/template-detail-view.component.html

```diff
@@ -0,0 +1,48 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<ng-container *ngIf="record">
+    <h2 id="template-name" class="pl-0" *ngIf="record.metadata.name">
+      {{ record.metadata.name }} <ng-container *ngIf="record.metadata.visibility && record.metadata.visibility === 'private'">
+        <i class="fa fa-lock text-secondary" aria-hidden="true"></i>
+      </ng-container>
+    </h2>
+    <dl class="row mb-0">
+      <!-- DESCRIPTION -->
+      <ng-container *ngIf="record.metadata.description">
+        <dt [ngClass]="dtCssClass">
+          <span class="label-title" translate>
+            Description
+          </span>
+        </dt>
+        <dd id="template-description" [ngClass]="ddCssClass">
+          {{ record.metadata.description }}
+        </dd>
+      </ng-container>
+      <!-- TYPE -->
+      <ng-container *ngIf="record.metadata.template_type">
+        <dt [ngClass]="dtCssClass">
+          <span class="label-title" translate>
+            Type
+          </span>
+        </dt>
+        <dd id="template-description" [ngClass]="ddCssClass">
+          {{ record.metadata.template_type }}
+        </dd>
+      </ng-container>
```

Comment:
Why not displaying others templates data like 'creator' and 'data' ? 
`data` should be use with `<pre>` tag to be pretty displayed.
If a template is public, I can't see a visual information about this fact.

Reply:
I had the instruction not to display data.
As in the brief view, an icon (lock) is displayed if the template is private

Reply:
The PO group validated the screenshots.